scripts/xe-backup-metadata

```diff
@@ -48,6 +48,28 @@ function usage {
     exit 1
 }
```

Comment:
this script is missing a generic `set -eu`: errors or uninitialised variables would not cause the script to stop, but it'll keep going, potentially executing the wrong things.